resources/js/views/settings/roles/View.vue

```diff
@@ -329,6 +383,8 @@ export default {
           }
 
           this.permissions[group].push(permission);
+
+          this.includedPermissionMap[permission.id] = permission.includedPermissions;
```

Comment:
Shouldn't we use here the set function?

Reply:
https://vuejs.org/v2/guide/reactivity.html#For-Objects

Reply:
Good catch! Took me a while to write a test that would fail with the current solution and works with the correct way.